ios/titanium.xcconfig

```diff
@@ -5,7 +5,7 @@
 //
 //
 
-TITANIUM_SDK_VERSION = 9.0.0.GA
+TITANIUM_SDK_VERSION = 9.0.0.v20200205142057
```

Comment:
Can you keep this at `9.0.0.GA`? Makes it easier for others to build the module from sources since it is setup with the latest GA release instead of a nightly build.